addons/calendar/controllers/main.py

```diff
@@ -5,61 +5,114 @@
 
 from odoo.http import request
 from odoo.tools.misc import get_lang
+from odoo.tools import consteq
 
 
 class CalendarController(http.Controller):
 
+    def fetch_attendee(self, id, attendee_id, token, state='accepted'):
```

Comment:
`state` seems to imply we want to fetch "accepted" attendees
Would rename `filter_state` or something like that

We don't use id so would just check if token is set or not

As we don't always use each parameter would make them optional defaulting to None